docs/Common.tex

```diff
@@ -1,14 +1,16 @@
 %% Common Parts
 
-\newcommand{\progname}{ProgName} % PUT YOUR PROGRAM NAME HERE
-\newcommand{\authname}{Team \#, Team Name
-\\ Student 1 name
-\\ Student 2 name
-\\ Student 3 name
-\\ Student 4 name} % AUTHOR NAMES                  
+\newcommand{\progname}{SFWRENG 4G06 Capstone Design Project} % PUT YOUR PROGRAM NAME HERE
+\newcommand{\projname}{MotionMingle} % project name
+\newcommand{\authname}{Team \#18, InfiniView-AI
+\\ Anhao Jiao
+\\ Kehao Huang
+\\ Qianlin Chen
+\\ Shu Qi
```

Comment:
Switch first name and last name

Reply:
Do we need to put the last name in front? [They](https://github.com/marlon4dashen/Hairesthetics/blob/main/docs/DevelopmentPlan/DevelopmentPlan.pdf) didn't do it.

Reply:
Oh it's Shu Qi. What a gamer.

Reply:
Corrected in baec17f